parlai/agents/hugging_face/hugging_face.py

```diff
@@ -17,7 +17,10 @@
     raise ImportError('Please run `pip install transformers`.')
 
 
-HF_VERSION = float('.'.join(transformers.__version__.split('.')[:2]))
+HF_VERSION = (
+    int(transformers.__version__.split('.')[0]),
+    int(transformers.__version__.split('.')[1]),
```

Comment:
nit (question): looking at their version history it looks like they only add letter like `rc` to the third digit. Is that always the case? Should we worry about `int` transformation here ever?

Reply:
i think we can worry about that later, if it becomes an issue. but good to know ahead of time